taxonomies/countries.txt

```diff
@@ -265,6 +265,7 @@ zh_min_nan:Sè-kài
 zh_yue:世界
 country_code_2:en:world
 wikidata:en:Q16502
+language_codes:en:so,af,ga,gn,ay,az,id,ms,bi,nb,bs,ca,sn,ny,cy,da,de,na,et,en,es,eu,fj,ch,mg,mg,fr,gv,sm,prs,gd,ho,hr,rw,rn,xh,zu,it,pl,mh,kl,sw,ht,lv,to,lt,ro,la,gl,lb,hu,mt,nl,nd,uz,pt,qu,mi,st,tn,ss,sk,sl,nr,fi,sv,sg,vi,ve,tk,tr,tl,ts,ku,is,cs,el,be,bg,tg,ky,mo,mk,mn,ru,sr,uk,kk,hy,he,ur,ar,fa,ps,dv,ne,hi,bn,ta,si,th,lo,dz,my,ka,ti,am,km,ja,zh,ko
```

Comment:
the test is failing because of this, because we use this property to add the no_index flag (and this is very important).
See Display.pm on line 773 

```perl
	if ((!($lc ~~ $country_languages{$cc})) or $subdomain =~ /^(ssl-)?api/) {
		# Use robots.txt with disallow: / for all agents
		$request_ref->{deny_all_robots_txt} = 1;

		if ($request_ref->{is_crawl_bot} eq 1) {
			$request_ref->{no_index} = 1;
		}
	}
```

So instead of adding this property, I propose that you add an exception in the code you modified in Display.pm with an hardcoded exception for "world".

Reply:
Note: the no_index is very important because otherwise we get hammered by indexing bots.